app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -12,9 +14,11 @@
 @Service
 @Transactional
 public class ProductService {
+    private final Mapper mapper;
     private final ProductRepository productRepository;
 
-    public ProductService(ProductRepository productRepository) {
+    public ProductService(Mapper dozerMapper,ProductRepository productRepository) {
```

Comment:
```suggestion
    public ProductService(Mapper dozerMapper, ProductRepository productRepository) {
```

여기 빈 칸이 들어가야 할 것 같아요. IDEA의 자동 정렬 사용을 습관하 하시면 좋을 것 같아요

Reply:
띄어쓰기, 빈칸 된 줄 알았는데 의외로 빈 곳이 있네요...